Sources/Testing/Events/Event.swift

```diff
@@ -306,6 +303,26 @@ extension Event {
   }
 }
 
+extension Configuration.EventHandlingOptions {
+  /// Determine whether the specified event should be handled according to the
+  /// options in this instance.
+  ///
+  /// - Parameters:
+  ///   - event: The event to consider handling.
+  ///
+  /// - Returns: Whether or not the event should be handled or suppressed.
+  fileprivate func shouldHandleEvent(_ event: borrowing Event) -> Bool {
+    switch event.kind {
+    case let .issueRecorded(issue):
+      issue.severity > .warning || isWarningIssueEventDeliveryEnabled
```

Comment:
Nitpick, but is this strictly the correct logic if we added some new severity later?